src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -220,13 +220,16 @@ private boolean runRepair() {
     Thread.currentThread().setName(clusterName + ":" + segment.getRunId() + ":" + segmentId);
 
     try (Timer.Context cxt = context.metricRegistry.timer(metricNameForRunRepair(segment)).time()) {
-      JmxProxy coordinator
-          = context.jmxConnectionFactory.connectAny(
+      JmxProxy coordinator =
+          context.jmxConnectionFactory.connectAny(
```

Comment:
can we keep the `=` on the newline.

in code style this clearly separates lines wrapped per parameter/argument versus operators.
this applies to the rest of the PR as well.

(the C* codebase also asks that operators carry when wrapping, it makes the code faster to read by providing this distinction)

Reply:
I created #556 to remove the need for nit-picking over carrying the operator when wrapping lines in the future.